src/Datadog.Trace.ClrProfiler.Managed/Integrations/Types/TypeNames.cs

```diff
@@ -0,0 +1,23 @@
+namespace Datadog.Trace.ClrProfiler
+{
+    internal class TypeNames
+    {
+        public const string Ignore = "_";
+
+        public const string Void = "System.Void";
+        public const string Object = "System.Object";
+        public const string Bool = "System.Boolean";
+
+        public const string Int8 = "System.Int8";
+        public const string Int16 = "System.Int8";
+        public const string Int32 = "System.Int32";
+        public const string Int64 = "System.Int64";
+
+        public const string UInt8 = "System.UInt8";
+        public const string UInt16 = "System.UInt8";
+        public const string UInt32 = "System.UInt32";
+        public const string UInt64 = "System.UInt64";
+
+        public const string Byte = Int8;
```

Comment:
```suggestion
        public const string Byte = UInt8;
```

Reply:
Shouldn't this be "System.Byte"?

Reply:
Also, you should be able to collapse UInt8 into Byte because `System.Byte` is the struct for an unsigned 8-bit integer.

Reply:
Probably. Maybe we should also drop `System.Int8` and `System.UInt8` since those don't exist as .NET types.